scripts/generate-builtin-functions.js

```diff
@@ -56,27 +56,27 @@ This file is auto-generated with "scripts/generate-builtin-functions.js"
 
 The built-in functions are divided into several categories:
     `.trim())
-    stream.write(categories.map(category => `\n[${category}](#${category.toLowerCase()}-functions)`).join(','))
-    stream.write(`.\nAll built-in functions are suffixed with \`!\`.`)
-    stream.write(`\nAll of the byte encoding use Big Endian byte order.`)
-    stream.write('\n\n')
+  stream.write(categories.map(category => `\n[${category}](#${category.toLowerCase()}-functions)`).join(','))
+  stream.write(`.\nAll built-in functions are suffixed with \`!\`.`)
+  stream.write(`\nAll of the byte encoding use Big Endian byte order.`)
+  stream.write('\n\n')
 }
 
 async function main() {
-    const url = 'https://raw.githubusercontent.com/alephium/alephium/master/protocol/src/main/resources/ralph-built-in-functions.json'
-    const response = await fetch(url)
-    const json = await response.json()
-    // const json = require('./builtin.json')
+  //const url = 'https://raw.githubusercontent.com/alephium/alephium/master/protocol/src/main/resources/ralph-built-in-functions.json'
+  //const response = await fetch(url)
+  //const json = await response.json()
+  // const json = require('./builtin.json')
 
-    const stream = fs.createWriteStream('./docs/ralph/built-in-functions.md')
-    const [categories, map] = extractCategory(json)
-    writeHeader(stream, categories)
-    categories.forEach(category => {
-        stream.write(`## ${category} Functions\n---\n`)
-        const functions = map.get(category)
-        functions.forEach(f => writeFunction(f, stream))
-    })
-    stream.end()
+  //  const stream = fs.createWriteStream('./docs/dapps/ralph/built-in-functions.md')
+  //const [categories, map] = extractCategory(json)
+  //writeHeader(stream, categories)
+  //categories.forEach(category => {
+  //  stream.write(`## ${category} Functions\n---\n`)
+  //  const functions = map.get(category)
+  //  functions.forEach(f => writeFunction(f, stream))
+  //})
+  //stream.end()
```

Comment:
Why this is commented out?

Reply:
Sorry, not intentional, did it because I was running `npm run build` offline. Reverted.